src/networkcloud/azext_networkcloud/operations/custom_properties.py

```diff
@@ -163,36 +164,37 @@ def _build_schema_operation_status_result_read(cls, _schema):
         _schema.start_time = cls._schema_operation_status_result_read.start_time
         _schema.status = cls._schema_operation_status_result_read.status
 
-    # Custom handling of response will print out the output head and the result URL
+    # Custom handling of response will display the output head and the result URL
     # it will also save files into output directory if provided
+    @staticmethod
     def _output(self, *args, **kwargs):
         args = self.ctx.args
         properties = self.ctx.vars.instance.properties
 
-        # Print the output head to the console
+        # Display the output head to the console
         if has_value(properties.outputHead):
```

Comment:
In python, we usually use snack case. And the aaz commands support snackcase property name of instance
```suggestion
        if has_value(properties.output_head):
```

Reply:
updated , thanks